### DISCUSSION TITLE
Suggested reading

### DISCUSSION THREAD

**Discussion by keithmcnulty** [rstudio-conf-2022/people-analytics-rstats#9] [Ideas]
This thread is to allow instructors, TAs, and participants to suggest further reading resources for the techniques covered in the workshop.  Please provide links where possible (eg website, Amazon purchase link, etc).

**Comment by alex-lopilato** [comment]
Survival Analysis Resources: 
 1. [Applied Survival Analysis Using R](https://www.amazon.com/Applied-Survival-Analysis-Using-Use/dp/331931243X/ref=sr_1_1?crid=X9KS6125TA4V&keywords=survival+analysis+in+r&qid=1658836228&s=books&sprefix=suvival+analysis+in+r%2Cstripbooks%2C84&sr=1-1)
 2. [Survival Analysis: A Self Learning Text](https://www.amazon.com/Survival-Analysis-Self-Learning-Statistics-Biology/dp/1441966455/ref=pd_bxgy_sccl_1/137-3555476-6681454?pd_rd_w=fQIri&content-id=amzn1.sym.7757a8b5-874e-4a67-9d85-54ed32f01737&pf_rd_p=7757a8b5-874e-4a67-9d85-54ed32f01737&pf_rd_r=T50PJHZBTTGXPBPFMKE7&pd_rd_wg=oswAa&pd_rd_r=f7dafb02-9053-4fdd-b7b2-f9566491e9c8&pd_rd_i=1441966455&psc=1)
 3. [Event History Modeling](https://www.amazon.com/Event-History-Modeling-Scientists-Analytical/dp/0521546737/ref=sr_1_1?crid=NOCNK0QL37JS&keywords=event+history+modeling&qid=1658836737&s=books&sprefix=event+history+modeling%2Cstripbooks%2C86&sr=1-1)

**Comment by keithmcnulty** [comment]
Foundational Statistics and Elementary Regression (R focused):  [Introduction to Modern Statistics](https://www.openintro.org/book/ims/), Cetinkaya-Rundel & Hardin

Linear Regression (Theoretical Basis, various software):  [Introduction to Linear Regression](https://www.amazon.com/Introduction-Regression-Analysis-Douglas-Montgomery/dp/0470542810/ref=asc_df_0470542810/?tag=hyprod-20&linkCode=df0&hvadid=312091458201&hvpos=&hvnetw=g&hvrand=6253364371108450634&hvpone=&hvptwo=&hvqmt=&hvdev=c&hvdvcmdl=&hvlocint=&hvlocphy=1027028&hvtargid=pla-433045377562&psc=1&tag=&ref=&adgrpid=63669393113&hvpone=&hvptwo=&hvadid=312091458201&hvpos=&hvnetw=g&hvrand=6253364371108450634&hvqmt=&hvdev=c&hvdvcmdl=&hvlocint=&hvlocphy=1027028&hvtargid=pla-433045377562), Douglas C Montgomery

Logistic regression (Theory and Examples):  [Applied Logistic Regression](https://www.amazon.com/Applied-Logistic-Regression-David-Hosmer/dp/0470582472/ref=d_pd_sbs_sccl_3_1/135-6859675-1186831?pd_rd_w=VmUMN&content-id=amzn1.sym.eb7ca553-5ebe-43f4-9e65-3cf60b79e964&pf_rd_p=eb7ca553-5ebe-43f4-9e65-3cf60b79e964&pf_rd_r=N03FCW3Z6AVAX8SK0KCF&pd_rd_wg=iQQhm&pd_rd_r=3571b993-9dbe-42f0-8a24-8788e52a0229&pd_rd_i=0470582472&psc=1), Hosmer & Lemeshow

Multinomial and Ordinal Regression (Theory and Examples):  [Categorical Data Analysis](https://www.amazon.com/Categorical-Data-Analysis-Alan-Agresti/dp/0470463635/ref=sr_1_4?crid=3KQHZOCO37A0P&keywords=Agresti&qid=1658838762&s=books&sprefix=agresti%2Cstripbooks%2C97&sr=1-4), Agresti

General Regression Modeling Strategies and Approaches (comes with a R Package `rms`):  [Regression Modeling Strategies](https://link.springer.com/book/10.1007/978-3-319-19425-7), Harrell

**Comment by liz-romero** [comment]
Regression and lots of other topics: [Data Analysis Using Regression and Multilevel/Hierarchical Models](https://www.amazon.com/Analysis-Regression-Multilevel-Hierarchical-Analytical-ebook/dp/B01LYX8AKU), Gelman

Network metrics: [A User’s Guide to Network Analysis in R](https://www.amazon.com/Users-Guide-Network-Analysis/dp/3319238825/), Luke

**Comment by keithmcnulty** [comment]
General network analysis including much more detail on today's content:  [Handbook of Graphs and Networks in People Analytics](https://ona-book.org)
